src/Persistence/Yaml/ControlObjectFactory.cs

```diff
@@ -89,8 +89,7 @@ internal void RestoreNestedTemplates(Control control)
                 continue;
 
             var nestedControl = _controlFactory.Create(Guid.NewGuid().ToString(), nestedTemplate);
-            if (control.Children == null)
-                control.Children = new List<Control>();
```

Comment:
It is easier to put break points when we have two lines

Reply:
this was a fix based on rule errors.
VS breakpoints can be set within a line.

Reply:
thanks for letting me know how to set breakpoints